README.md

```diff
@@ -1,5 +1,11 @@
 # Actividades de AsturiasHacking
 
+<p align="center">
```

Comment:
And now it looks less "important" xD

Maybe the other way around? I don't know anymore :(